server/src/QRServer/server.py

```diff
@@ -120,6 +124,11 @@ async def task_remover():
     async def start_tasks(self):
         self.start_task("Discord Logger", discord_logger.get_daemon_task(self.config))
         self.start_task("Cron Logger", self._cron_logger())
+
+        if self.config.discord_bot_enabled:
```

Comment:
When I ran tests this evaluated to `True`, even though I did not create any config, it may be worth checking out

Reply:
resolved